feature/bgp/otg_tests/base_bgp_session_parameters/README.md

```diff
@@ -48,48 +48,21 @@ Test the normal session establishment and termination:
     *   Explicit holdtime interval and keepalive interval.
     *   Explicit connect retry interval.
 
-## Config Parameter coverage
-
-*   For prefix:
-
-    *   /network-instances/network-instance/protocols/protocol/bgp/global
-
-*   For Parameters:
-
-    *   config/as
-    *   config/router-id
-    *   config/peer-as
-    *   config/local-as
-    *   config/description
-    *   timers/config/hold-time
-    *   timers/config/keepalive-interval
-    *   timers/config/minimum-route-advertisement-interval
-
-*   For prefixes:    
-
-    *   /network-instances/network-instance/protocols/protocol/bgp/peer-groups/peer-group
-    *   /network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor
-    
-## Telemetry Parameter coverage   
-    
-*   For prefix:
-    
-    *   /network-instances/network-instance/protocols/protocol/bgp/
-
-*   For Parameters:
-
-    *   state/last-established
-    *   state/messages/received/NOTIFICATION
-    *   state/negotiated-hold-time
-    *   state/supported-capabilities
-
-## Protocol/RPC Parameter coverage
-
-*   BGP
-    
-    *   OPEN
-    
-        *   Version
-        *   My Autonomous System
-        *   BGP Identifier
-        *   Hold Time
+## OpenConfig Path and RPC Coverage
+```yaml
+paths:
```

Comment:
typically, best practice is to only make mechnical changes when doing a large scale move, so that we don't have to review the contents of each file. subsequent clean-up can be done in individual changes that are easier to review.

Reply:
the README CI check is triggered for README which are touched in a PR.  It requires that the path list is in the correct header and contains valid OC paths.  Quite a few of these readmes to be moved did not meet this.   (Thank you very much for fixing all this tech debt @self-maurya )

The path content in the readme did not change from my read of this.